packages/shared/src/components/MainLayout.tsx

```diff
@@ -225,12 +222,9 @@ function MainLayoutComponent({
       />
       <main
         className={classNames(
-          'flex flex-col',
+          'flex flex-col tablet:pl-16 laptop:pl-11',
           className,
-          !isScreenCentered && sidebarExpanded
-            ? 'laptop:pl-60'
-            : 'laptop:pl-11',
-          isNewMobileLayout && 'tablet:pl-16',
+          !isScreenCentered && sidebarExpanded && 'laptop:!pl-60',
```

Comment:
Have no strong opinion here since it looks more understandable with the inline changes you made above, but just not a fan of `!`. Either way should be good 🚀

Reply:
No strong opinion either, just thought this way it was more clear